### ISSUE TITLE
CI failing due to no vobject test code

### ISSUE THREAD

**Issue by phil-davis** [sabre-io/dav#1327]
See demo PR #1326 

https://travis-ci.org/github/sabre-io/dav/jobs/757531864
```
Test directory "tests/../vendor/sabre/vobject/tests/VObject" not found
```

This is because of PR https://github.com/sabre-io/vobject/pull/525 - the test artifacts (and other "rubbish") are not bundled in the release package any more since `sabre/vobject` release 4.3.4 today.

Why do we run the unit tests of dependencies in the CI of `sabre/dav` ?

**Comment by phil-davis** [comment]
Possible "fix" in PR #1328 - "deleted code is debugged code" :)

**Comment by phil-davis** [comment]
Fixed in #1329
